gatsby/installing.rst

```diff
@@ -33,13 +33,13 @@ The CLI allows to initialize a project:
    gatsby new hello-world
 
 This command says gatsby-cli to create a new Gatsby project with name `hello-world` with a basic default file structure.
+
 There are several boilerplates created by the community that allows to easily bootstrap an application for different use-cases.
```

Comment:
"allow a developer to bootstrap"

Reply:
same with "several" this makes it wordy without adding value